code/modules/mob/living/simple_animal/bot/bot.dm

```diff
@@ -464,6 +464,13 @@ Pass the desired type path itself, declaring a temporary var beforehand is not r
 			continue //The current element failed assessment, move on to the next.
 		return final_result
 
+/mob/living/simple_animal/bot/proc/check_bot(targ)
+	var/turf/T = get_turf(targ)
+	if(T)
+		for(var/C in T.contents)
+			if(istype(C,type) && (C != src)) //Is there another bot there already? If so, let's skip it so we dont all atack on top of eachother.
+				return TRUE //Let's abort if we find a bot so we dont have to keep rechecking
```

Comment:
```suggestion
	if(!T)
		return

	for(var/C in T.contents)
		if(istype(C, type) && C != src) //Is there another bot there already? If so, let's skip it so we dont all atack on top of eachother.
			return TRUE //Let's abort if we find a bot so we dont have to keep rechecking
```
Early returns are preferred. Also, spacing and the ()'s around the != check are not required.
Might also want to filter on the bot type in the for loop so it saves some processing power. So like this:
```
for(var/mob/living/simple_animal/bot/B in T.contents)
```